src/data/finset/lattice.lean

```diff
@@ -820,6 +820,24 @@ theorem le_max_of_mem {s : finset α} {a b : α} (h₁ : a ∈ s) (h₂ : s.max
 by rcases @le_sup (with_bot α) _ _ _ _ _ _ h₁ _ rfl with ⟨b', hb, ab⟩;
    cases h₂.symm.trans hb; assumption
 
+lemma mem_le_max {a : α} {s : finset α} (as : a ∈ s) :
+  ↑a ≤ s.max :=
+(le_fold_max _).mpr (or.inr ⟨a, as, rfl.le⟩)
```

Comment:
I would assume this has a one-term proof using the corresponding statement about sup

Reply:
I could not find the corresponding statement about sup, so I added it.  Due to some decidability issues, though, I added two:
* `le_sup_of_mem`, which seems more natural, but wanted `[decidable_eq α]`
* `le_sup_of_mem_set`, which to me seems to be saying almost the same thing, except that this one works to prove `coe_le_max_of_mem`.

Feel free to suggest better versions/remove for either one of them!